### DISCUSSION TITLE
Maui .Net 8 Button Click Animation Missing

### DISCUSSION THREAD

**Discussion by welcometochristown** [dotnet/maui#17599] [Q&A]
Why does the button click animation not work in my Android app .Net8 for Maui, but does work in .Net7?

Using the default project for both, only in .Net7 can I see the little fade animation when I press the button.

Was there a change in .Net 8 that I have adjust to get that animation back? I would like this feedback to the user.

I tried updating to .Net8 rc1 but the issue still exists.

.net7
![Record_2023-09-23-12-16-29](https://github.com/dotnet/maui/assets/17158437/02bf0055-bd42-4d04-9cba-945a393659e5)

.net8
![Record_2023-09-23-12-17-06](https://github.com/dotnet/maui/assets/17158437/96f25308-a7fd-4e68-84d1-9e2d4c16960b)

**Comment by Strypper** [comment]
quALitY 🤡🤡🤡

**Comment by Engisan** [comment]
Hey, did you find a way to get it to work? I face the same issue and this is annoying :/

**Comment by Ghostbird** [comment]
We added something like this:
(Note: `TapAnimation` is our own animation)

```
public override WithTapAnimation<TButton>(this TButton self)
where TButton : Button
{
  self.Behaviors.Add(
   new AnimationBehavior
    {
      EventName = nameof(Button.Clicked),
      AnimationType = new TapAnimation()
    }
  );
  return self;
}
```

Note that you MUST specify `EventName` or you'll run into the fact that `Button`s will break on iOS in MAUI 8 if anything tries to use `TapGestureRecognizer` on them, which `AnimationBehavior` does by default. (see #19099 )
